### PR TITLE
Add CZI employees method to feature

### PR DESCRIPTION
**Pull request by gregdingle**
# Description

See comments in code. 

# Tests

* run in localhost and see changes to user objects

### CODE REVIEW

File: app/models/user.rb [hunk 1]
```diff
@@ -79,6 +79,17 @@ def remove_allowed_feature(feature)
...
   end
 
+  # This is a convenience function for the common case of adding all team
+  # members, test accounts, contractors and beta testers to a new feature before
+  # general release. IMPORTANT NOTE: it assumes that only valid personnel have
+  # a @chanzuckerberg address. This method is intended to be called from the rails
+  # console.
+  def add_allowed_feature_for_all_czi(feature)
+    # covers contractors and regular employees
+    users = User.where("email like '%@%chanzuckerberg.com'")
```
Comment (jshoe) [inline app/models/user.rb]: could we merge `@contractor.chanzuckerberg.com` and `@chanzuckerberg.com` instead? Test firm might report something like this

Comment (gregdingle) [reply app/models/user.rb]: sure that would be safer

File: app/assets/src/components/views/samples/MetadataUploadModal/Instructions.jsx [hunk 2]
```diff
@@ -43,6 +43,19 @@ class UploadInstructions extends React.Component {
               >
                 CSV template.
               </a>
+              <p>
+                {`If your entered Host Organism does not match a supported host
+              genome, we will only subtract out ERCCs and the Human genome. You can
+              read more about how to request a new genome to be added to IDseq `}
+                <a
+                  href="https://chanzuckerberg.zendesk.com/hc/en-us/articles/360034790814-How-do-I-request-a-new-host-genome-be-added-to-IDseq-"
```
Comment (tfrcarvalho) [inline app/assets/src/components/views/samples/MetadataUploadModal/Instructions.jsx]: Can we remove the redundancy? Define the description and url as constants somewhere?

Comment (tfrcarvalho) [reply app/assets/src/components/views/samples/MetadataUploadModal/Instructions.jsx]: Wait is this part of this PR?

**Comment by jshoe** [comment]
^ I think the last commit went to the wrong branch!

File: app/assets/src/components/views/samples/MetadataUploadModal/Instructions.jsx (hunk 2 above)
Comment (gregdingle) [reply app/assets/src/components/views/samples/MetadataUploadModal/Instructions.jsx]: Sorry wrong branch. I have two in progress now. One min please.

File: app/models/user.rb [hunk 3]
```diff
@@ -79,6 +79,19 @@ def remove_allowed_feature(feature)
     end
   end
 
+  # This is a convenience function for the common case of adding all team
+  # members, test accounts, contractors and beta testers to a new feature before
+  # general release. IMPORTANT NOTE: it assumes that only valid personnel have
+  # a @chanzuckerberg address. This method is intended to be called from the rails
+  # console.
+  def self.add_allowed_feature_for_all_czi(feature)
```
Comment (tfrcarvalho) [inline app/models/user.rb]: This kind of custom actions make more sense to me as a rake task.

Comment (gregdingle) [reply app/models/user.rb]: true. and would that include the other `add_allowed_feature` method?

Comment (tfrcarvalho) [reply app/models/user.rb]: No, what i find very specific is a the email parameter... E.g. eventually we might have a rake task for list of users, or biohub emails...

**Comment by gregdingle** [comment]
I don't want to invest the time to make a rake task right now so I'm just going to close this PR. Anyone is welcome to pick up the idea. I only posted this PR because I had just run the exact commands in the console.

**Pull request closed without merging**